app/src/fdroid/java/com/eventyay/organizer/core/event/create/LocationPicker.java

```diff
@@ -0,0 +1,39 @@
+package com.eventyay.organizer.core.event.create;
+
+import android.annotation.SuppressLint;
+import android.app.Activity;
+import android.content.Intent;
+
+public class LocationPicker {
+
+    private final double DEMO_VALUE = 1;
+
+    public boolean launchPicker(Activity activity) {
+        //do nothing
+        return false;
+    }
+
+    @SuppressLint("RestrictedApi")
+    public Location getPlace(Activity activity, Intent data) {
+        return new Location();
+    }
+
+    public boolean shouldShowLocationLayout() {
+        return true;
+    }
+
+    public class Location {
+
+        public double getLatitude() {
+            return DEMO_VALUE;
+        }
+
+        public double getLongitude() {
+            return DEMO_VALUE;
+        }
+
+        public CharSequence getAddress() {
+            return null;
+        }
+    }
```

Comment:
This `Location` class can be shared

That was the notion of creating it, Add a constructor which takes lat, long, and `String` address

There should be only one class definition.

Thanks for this. Everything else looks good. Amazing Job!